### ISSUE TITLE
Google sign in fails to find elements when run headless 

### ISSUE THREAD

**Issue by qualityshepherd** [DevExpress/testcafe#4434]
## Steps:
1. given a Testcafe project or Testcafe is installed globally
1. save the code below to a file (eg. `test.js`)
1. run headlessly with latest (1.6.0) version of Testcafe and either Chrome (78.0.3904.70) or Firefox (70.0.0)
1. eg. `./node_modules/.bin/testcafe firefox:headless tests/test.js`

## Results:
The test fails trying to locate Google's sign in email input (`#identifierId`) and throws an error:
` 1) The specified selector does not match any element in the DOM tree. > | Selector('#identifierId')`

## Impact:
This does _not_ occur when run non-headless in chrome or ff. 
node v8.16.1
Mac OS X 10.14.0 (also occurs on Ubuntu CI)

## Test code
```
import { Selector } from "testcafe";

// google signin
const gmailUrl = 'https://accounts.google.com/signin/v2/';
const googleEmail = Selector('#identifierId');
const googleEmailNextBtn = Selector('#identifierNext');
const googlePassword = Selector('[name="password"]');
const googlePasswordNextBtn = Selector('#passwordNext');
const email = 'fake@gmail.com';
const password = 'password';

fixture`Headless Testcafe Tests`;

test('should sign in via google', async (t) => {
  await t
    .navigateTo(gmailUrl)
    // fails here with error: "The specified selector does not match any element in the DOM tree.""
    .typeText(googleEmail, email)
    .click(googleEmailNextBtn)
    .typeText(googlePassword, password)
    .click(googlePasswordNextBtn);
});
```

**Comment by AlexKamaev** [comment]
I was able to reproduce the issue.
It looks like in headless mode Chrome opens another page that does not have the required element. Could you please enable the [take-on-fails](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#takeonfails) option and check if the page differs in headless mode?

**Comment by qualityshepherd** [comment]
Yup... chrome and firefox are 404ing in headless mode... 

![image](https://user-images.githubusercontent.com/3298906/68419620-d412b600-0157-11ea-8634-69d87dc276ef.png)

**Comment by AndreyBelym** [comment]
It looks like that we can just remove the trailing slash in the sign-in URL to avoid the issue: `https://accounts.google.com/signin/v2`.

I've found that the problem with the 404 page was caused by the trailing slash in the URL. I was able to reproduce it both in headless and regular Chrome. I don't know why behaviour was different in your case. Maybe Google performed some sort of A/B testing. But for me it's clear that the problem is on the Google side.
